README.md

```diff
@@ -117,6 +119,7 @@ Yields `UsageInfo` objects as described below.
 {
   browsers: ['ie >= 8', '> 1%'] // an autoprefixer-like array of browsers.
   ignore: ['rem'], // an optional array of features to ignore
+  ignoreFiles: ['**/normalize.css'], // an optional array of file globs to ignore
```

Comment:
Maybe add a note that these globs refer to the original source file path, using the source map?